LeaderboardBackend/Controllers/Annotations/Conventions.cs

```diff
@@ -6,6 +6,7 @@ namespace LeaderboardBackend.Controllers.Annotations;
 public static class Conventions
 {
 	[ProducesResponseType(StatusCodes.Status200OK)]
+	[ProducesResponseType(StatusCodes.Status400BadRequest)]
 	[ProducesResponseType(StatusCodes.Status404NotFound)]
```

Comment:
Adding an attribute here will mean that ASP.NET expects doc comments for every action that uses this convention. We sure we want this?

Reply:
Guess we could make a new convention instead?

Reply:
I'm not sure if it's worth creating a new convention. Guess we could add one for now? Could always remove it if needed

Reply:
Decided to leave this, because theoretically all routes can produce 400 and the docs get generated anyway